src/backend/impl/src/lib.rs

```diff
@@ -1,6 +1,43 @@
+#![deny(clippy::all)]
+
+use backend_api::*;
+use candid::export_service;
 use ic_cdk::*;
 
-#[query]
-fn say_hello() -> String {
-    "Hello, world!".to_string()
+mod controllers;
+mod mappings;
+mod repositories;
+mod services;
+mod system_api;
+
+// https://github.com/la10736/rstest/tree/master/rstest_reuse#cavelets
+#[cfg(test)]
+#[allow(clippy::single_component_path_imports)]
+use rstest_reuse;
+
+#[cfg(test)]
+mod fixtures;
+
+export_service!();
```

Comment:
These next lines are a nice unit test to make sure the Rust interface is compatible with the Candid interface.